frontend/src/components/project/Buttons/LikeButton.js

```diff
@@ -113,13 +135,15 @@ export default function LikeButton({
         <Button
           onClick={handleToggleLikeProject}
           variant="contained"
-          startIcon={<LikeIcon size={20} color={isUserLiking ? "earth" : "white"} />}
+          startIcon={<ButtonIcon icon="like" size={26} color={isUserLiking ? "earth" : "white"} />}
           color={isUserLiking ? "secondary" : "primary"}
           disabled={likingChangePending}
           className={classes.largeLikeButton}
         >
-          {likingChangePending && <CircularProgress size={13} className={classes.fabProgress} />}
-          {isUserLiking ? texts.liked : texts.like}
+          <div className={classes.buttonLabel}>
+            {likingChangePending && <CircularProgress size={20} className={classes.fabProgress} />}
```

Comment:
```suggestion
            <CircularProgress size={20} className={`${classes.fabProgress} ${!likingChangePending && classes.hidden}`} />
```

For me the loading animation actually doesn't show load. It only works when I directly set likingChangePending to true. It seems like the time isn't enough to load the SVG. Therefore this workaround could be a good solution: actually rendering but hiding the CircularProgress directly on page load.